src/fairseq2/models/wav2vec2/builder.py

```diff
@@ -139,6 +139,9 @@ class Wav2Vec2EncoderConfig:
     depthwise_conv_kernel_size: int
     """The kernel size of depthwise convolutions in Conformer blocks."""
 
+    conv_norm_type: Literal["batch_norm", "layer_norm"]
+    """The type of norm layer in the Conformer convolution module."""
```

Comment:
Very nit: I would remove the term "layer" from this docstr to avoid confusion between batch and layer normalization. I would just write "The type of normalization to use in Conformer convolution modules".